core/trino-main/src/main/java/io/trino/operator/exchange/LocalExchange.java

```diff
@@ -88,7 +88,7 @@ public LocalExchange(
             int defaultConcurrency,
```

Comment:
`Refactor LocalExchange` -> `Pass partition channel types directly to LocalExchange`